solution/src/main/openapi/solution.yaml

```diff
@@ -1599,7 +1619,8 @@ components:
         description: A Brewery Supplier Solution Model which simulate stock, production, transport and customer satisfaction
         repository: brewery_solution
         version: "1.0.0"
-        ownerId: "1"
+        creationDate: 1713276739000
```

Comment:
This should be removed and `creationInfo` / `updateInfo` should be added instead